common/src/main/scala/org/neo4j/spark/service/SchemaService.scala

```diff
@@ -666,7 +666,8 @@ class SchemaService(
           val quotedProps = props
             .map(prop => s"${Neo4jUtil.NODE_ALIAS}.${prop.quote()}")
             .mkString(", ")
-          val isNeo4j5 = neo4jVersion().versions(0).startsWith("5.")
+          val isNeo4j5 =
```

Comment:
should we just revert this check instead? like if it is "4." then do the special logic but assume later versions instead?

Reply:
fixed now